PWGHF/D2H/Tasks/taskB0Reduced.cxx

```diff
@@ -351,6 +437,76 @@ struct HfTaskB0Reduced {
         }
       }
     }
+    if (fillTree) {
```

Comment:
General comment (take it more as a suggestion for a future PR): you can consider to unify the `fillCand` and `fillCandMcReco` functions in a unique `template` function, and manage with a `constexpr bool IS_MC` variable the decision whether to fill data or MC objects (and whether to do MC-specific operations).
This might help reducing code duplications and be helpful in case you develop new features (for example, as it is now if you add a new variable in the tree you must remember to add the column filling twice).

Reply:
Hi @mfaggin ! Thanks a lot for the comments, I am implementing them.

Yes, I fully agree, I also thought about this template improvement, and we'll indeed keep it for a future PR :)